pkg/api/validation/validation.go

```diff
@@ -3517,3 +3530,17 @@ func isValidHostnamesMap(serializedPodHostNames string) bool {
 	}
 	return true
 }
+
+func sysctlIntersection(a []api.Sysctl, b []api.Sysctl) []api.Sysctl {
+	lookup := make(map[string]struct{}, len(a))
```

Comment:
This line frightens me: https://github.com/sttts/kubernetes/blob/836ac6e4037c2ff0c9994ced9526b6a18a451486/vendor/github.com/heketi/heketi/pkg/utils/stringset.go#L40-L40

O(n*n*log(n))